tests/prefetch_related/tests.py

```diff
@@ -978,6 +978,51 @@ def test_nested_prefetch_related_with_duplicate_prefetcher(self):
         with self.assertNumQueries(5):
             self.traverse_qs(list(houses), [["occupants", "houses", "main_room"]])
 
+    def test_nested_prefetch_related_with_duplicate_prefetch_and_depth(self):
+        """
+        In the case of a defined to_attr, the order should not have an impact.
+
+        Ticket: #34791
+        """
```

Comment:
You can drop the test description, the test name and the commit itself which will link back to the ticket should be enough.